lib/CXGN/List/Validate/Plugin/Locations.pm

```diff
@@ -15,16 +15,66 @@ sub validate {
 
 #    print STDERR "LIST: ".Data::Dumper::Dumper($list);
 
-    my @missing = ();
-    foreach my $term (@$list) { 
+   my $location_code_type_id = SGN::Model::Cvterm->get_cvterm_row($schema, 'abbreviation', 'geolocation_property')->cvterm_id();
```

Comment:
Not sure if this is used in the brapi endpoints and if it would have any impact?

Reply:
Not used in brapi endpoints